spec/lib/salus/scanners/osv/gradle_osv_spec.rb

```diff
@@ -80,15 +80,30 @@ def stub_req_with_valid_response
         expect(scanner.report.to_h.fetch(:passed)).to eq(false)
       end
 
+      it 'should fail when vulnerable dependencies are found in multi build project' do
+        repo = Salus::Repo.new(File.join(fixture_path, 'multi_build_project'))
+        config_data = YAML.load_file(
+          File.join(fixture_path, 'multi_build_project/salus.yaml')
+        )
+        scanner = Salus::Scanners::OSV::GradleOSV.new(
+          repository: repo, config: config_data["scanner_configs"]["GradleOSV"]
+        )
+        stub_req_with_valid_response
+        scanner.run
+
+        expect(scanner.report.to_h.fetch(:passed)).to eq(false)
+      end
+
       it 'should pass when vulnerable dependencies found in build.gradle'\
-       'have exceptions configured' do
+       ' have exceptions configured' do
```

Comment:
very minor - not sure if this space at the beginning was intentional.

Reply:
Ohh i just added because if otherwise it joined the word,  build.gradlehave